Packs/Claroty/Integrations/Claroty/Claroty_test.py

```diff
@@ -102,7 +102,7 @@ def _create_client(mocker, requests_mock, request_url, response_json, request_ty
             'password': 'omgSecretsWow',
         },
         'url': 'https://website.com',
-        'fetch_time': '7 days'
+        'first_fetch': '7 days'
```

Comment:
```suggestion
        'fetch_time': '7 days'
```
Let's revert this as well